ivy/functional/ivy/creation.py

```diff
@@ -331,6 +331,9 @@ def zeros(
            [0., 0., 0., 0., 0.],
            [0., 0., 0., 0., 0.]])
 
+    >>> x = ivy.zeros(5)
```

Comment:
There is already an example of how this function works above. Not sure this example is needed.

Reply:
I added this as it is mentioned in Docstring Examples to cover all possible variants. There was one example which mentioned a tuple. Hence, I added a number